src/explore-education-statistics-admin/src/pages/release/data/components/__tests__/ReleaseDataGuidanceSection.test.tsx

```diff
@@ -259,7 +259,7 @@ describe('ReleaseDataGuidanceSection', () => {
 
       expect(subject2Footnotes).toHaveLength(1);
       expect(subject2Footnotes[0]).toHaveTextContent('Footnote 3');
-    });
+    }, 20000);
```

Comment:
There's a few timeouts we've added across the changes. Do we actually need these now with the faster test runners?

Reply:
I've played around with the values and removed the majority except for one instance in `ChartBuilder.test.tsx`